### PR TITLE
Bugfix/translations

### PR DESCRIPTION
**Pull request by michaeljcollinsuk**

### CODE REVIEW

File: wcivf/apps/elections/templates/elections/includes/_polling_place.html [hunk 1]
```diff
@@ -23,28 +23,34 @@ <h2 id='where'>
...
             {% else %}
                 {% if ballots_today and not multiple_city_of_london_elections_today %}
-                    <strong>Polling stations are open from {{ ballots_today.0.election.polls_open|time:"ga" }} till {{ ballots_today.0.election.polls_close|time:"ga" }} today.</strong>
+                    {% blocktrans trimmed with open_time=ballots_today.0.election.polls_open|time:"ga" close_time=ballots_today.0.election.polls_close|time:"ga" %}
+                        <strong>Polling stations are open from {{ open_time }} till {{ close_time }} today.</strong>
+                    {% endblocktrans %}
                 {% endif %}
                 {% if polling_station.addresses %}
-                    <p>Your polling station in {{ postcode }} depends on your address. <a href="https://wheredoivote.co.uk/postcode/{{ postcode }}/">Check the correct polling station for your address &raquo;</a></p>
+                    {% blocktrans trimmed with postcode=postcode %}
```
Comment (symroe) [inline wcivf/apps/elections/templates/elections/includes/_polling_place.html:38]: I think `postcode=postcode` is redundant?

**Review by symroe — APPROVED** [review]

File: wcivf/apps/elections/templates/elections/includes/_polling_place.html (hunk 1 above)
Comment (michaeljcollinsuk) [reply wcivf/apps/elections/templates/elections/includes/_polling_place.html:38]: Ah so vars can be read in the blocks, just not vars read from the an object? I'll remove

Comment (michaeljcollinsuk) [reply wcivf/apps/elections/templates/elections/includes/_polling_place.html]: Double checked, you were right so I have removed

Comment (symroe) [reply wcivf/apps/elections/templates/elections/includes/_polling_place.html]: Yeah, it's just complex vars that don't work. Using a filter on `postcode` wouldn't work either

**Pull request merged**